### PR TITLE
Add RTC clock menu

### PR DESCRIPTION
**Pull request by mkopec**
Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

File: utils/rtc_clock_menu.c [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+ */
+
+#include <libpayload.h>
+#include <curses.h>
+
+static void print_rtc_clock_menu(struct tm *t) {
```
Comment (miczyg1) [inline utils/rtc_clock_menu.c:21]:
```suggestion
static void print_rtc_clock_menu(struct tm *t)
{
```
Brackets of functions should start in new line. Please fix for all other functions

File: sortbootorder.c [hunk 2]
```diff
@@ -493,6 +497,7 @@ static void show_boot_device_list(char buffer[MAX_DEVICES][MAX_LENGTH],
 #endif
 	printf("  w Enable BIOS write protect - Currently %s\n",
 		(spi_wp_toggle) ? "Enabled" : "Disabled");
+	printf("  z Clock menu\n")
```
Comment (miczyg1) [inline sortbootorder.c:500]:
```suggestion
	printf("  z Clock menu\n");
```

Was it even build tested?

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: utils/rtc_clock_menu.c [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+		t->tm_mon = mon - 1;
+	else
+		printf("Please enter a month between 1 and 12\n");
+}
+
+static void handle_day_command(struct tm *t, char *command) {
+	int day;
+
+	day = strtoul(command+2, NULL, 10);
+	if (day >= 1 && day <= 31)
```
Comment (miczyg1) [inline utils/rtc_clock_menu.c:60]: Oooh, can we set 31st of February? Interesting...

File: utils/rtc_clock_menu.c (hunk 1 above)
Comment (mkopec) [reply utils/rtc_clock_menu.c]: Fixed

File: utils/rtc_clock_menu.c (hunk 3 above)
Comment (mkopec) [reply utils/rtc_clock_menu.c]: Fixed.

Comment (miczyg1) [reply utils/rtc_clock_menu.c]: Yes, any OS would fix it and sync the time via NTP or something... But... are we sure any software is capable of that? Think about the issue reported on GitHub.

**Pull request merged**